data/data-pipeline/data_pipeline/ipython/scoring_comparison.ipynb

```diff
@@ -336,6 +359,19 @@
     "merged_df.head()"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "462d652e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Check that this in alignment with Pg 151. index number 50 for Michigan EJScreen\n",
```

Comment:
I'm not sure I understand what's going on here. Should we delete this cell to keep the notebook clean, or do you expect everyone running this notebook in the future to check this?

Reply:
Good question. I was coming at it from the perspective a new individual - representing Michigan (or any other state for that matter) - looking at the output and questioning whether the tracts that were ranked as burdened in the published white-paper did indeed match what was provided in the output of the ETL pipeline.

Reply:
Ok. I think I'd vote that we remove it: we don't do this check in this notebook for any other sources, and if we did there'd be 20 of these cells. It's extremely valuable to do these kinds of checks when developing ETL / analytics code, but I'm not sure we want them in the final product that will be used by many other data scientists / engineers.